compiler/x/runtime/VirtualGuardRuntime.cpp

```diff
@@ -47,15 +48,15 @@ extern "C" void _patchVirtualGuard(uint8_t *locationAddr, uint8_t *destinationAd
       // Self-loop
       *(uint16_t*)locationAddr = 0xfeeb;
 
-      _patchingFence16(locationAddr);
```

Comment:
Can you now delete the assembly proc `_patchingFence16` ?

Reply:
Probably not right now, as downstream projects may still use it. However, I'll clean it up as soon as downstream projects adopt the new intrinsic-based version.